chat-commands.js

```diff
@@ -2023,16 +2023,17 @@ exports.commands = {
 		let targetUser = this.targetUser;
 
 		if (!targetUser) {
-			return this.errorReply("User '" + this.targetUsername + "' not found.");
+			return this.errorReply(`User '${this.targetUsername}' not found.`);
 		}
 		if (!this.can('forcerename', targetUser)) return false;
-		if (targetUser.namelocked) return this.errorReply("User '" + target + "' is already namelocked.");
+		if (targetUser.namelocked) return this.errorReply(`User '${target}' is already namelocked.`);
 
-		this.addModCommand("" + targetUser.name + " was namelocked by " + user.name + "." + (reason ? " (" + reason + ")" : ""));
-		this.globalModlog("NAMELOCK", targetUser, " by " + user.name + (reason ? ": " + reason : ""));
+		if (!reason) reason = '';
```

Comment:
??

``reason`` is either a string or ``undefined`` here, so by doing this, if it's ``undefined``, it will be ``''``, which is also a falsey value but looks fine in the UI - this prevents me from having to use a ternary operator below